content/millennium/r4/foundation/conformance/structure-definition.md

```diff
@@ -145,6 +147,7 @@ Authorization is not required.
 [`financial-transaction-tendered-amount`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-tendered-amount?_format=json
 [`financial-transaction-type`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-type?_format=json
 [`formatted-text`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/formatted-text?_format=json
+[`national-drug-product`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/national-drug-product?_format=json
 [`modifier`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/modifier?_format=json
```

Comment:
nit: should go after modifier

Reply:
fixed -https://github.com/cerner/fhir.cerner.com/pull/683/commits/fc375ce65a3a2c97ad6800ce3e82f68e0377e556